app/models/invoice.rb

```diff
@@ -4,14 +4,41 @@
 #
 #  id                :integer          not null, primary key
 #  stripe_invoice_id :string
-#  brand_id          :integer
+#  brand_id          :integer          not null
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
 #  paid_at           :datetime
 #  amount            :integer
-#  data              :string
+#  data              :jsonb            not null
 #
 
 class Invoice < ActiveRecord::Base
   belongs_to :brand
+
+  def normalize_data
+    return unless self.data.key?('data') && self.data.key?('object')
```

Comment:
In the stripe webhook invoices controller, we [strip out](https://github.com/project-signal/project-signal/blob/master/app/controllers/webhooks/stripe/invoices_controller.rb#L20) the `data` and `object` keys. I don't think we actually need to do this check. We shouldn't be saving the data of the invoice object incorrectly.